core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -393,12 +451,47 @@ public AdvancedTlsSettings advancedTls() {
             return new AdvancedTlsSettings();
         }
 
+        /**
+         * Set the maximum number of rows that can be buffered locally before they are automatically sent to a server.
+         * <br>
+         * This is only used when communicating over HTTP transport, and it's illegal to call this method when
+         * communicating over TCP transport.
+         * <br>
+         * The Sender will automatically flush the buffer when it reaches this limit. You must make sure that
+         * the buffer is flushed before it reaches the maximum capacity. Otherwise, the Sender will throw an exception
+         * when you try to add more data to the buffer.
```

Comment:
This is confusing. Does it:
* Automatically flush and clear the buffer? or
* Error out if the buffer breaches the line count?

The paragraph describes both behaviours, yet these are mutually exclusive (or I'm just downright confused).

Reply:
good catch. how about this?
```
The Sender will automatically flush the buffer when it reaches this limit. You must make sure that
the buffer has sufficient capacity to accommodate all locally buffered data. Otherwise, the Sender
will throw an exception.
```

Reply:
link to`@see #maxBufferCapacity(int)`

Reply:
Oh I see. Maybe simpler:

```
The Sender will automatically flush the buffer after a row breaches this limit.
Ensure this limit is set lower than @see #maxBufferCapacity().
```